servicebuilder-activemq/src/main/java/no/obos/util/servicebuilder/mq/ActiveMqListener.java

```diff
@@ -110,6 +111,24 @@ private void handleMessage(MessageHandler handler, Message message, Session sess
         }
     }
 
+    static String truncateMessageForLogging(String text) {
+        if (StringUtils.isEmpty(text) || text.length() <= MAX_LOGGED_PER_MESSAGE) {
+            return text;
+        }
+        try {
+            log.info("Truncating message for logging...");
+            String truncatedText = text.substring(0, MAX_LOGGED_PER_MESSAGE);
+            String dataKey = "\"data\":";
+            if (truncatedText.contains(dataKey)) {
+                return truncatedText.substring(0, truncatedText.indexOf(dataKey)) + dataKey + "\"...\"}";
```

Comment:
Kan ha mer beskrivende innhold i data, men ikke så farlig.

```suggestion
                return truncatedText.substring(0, truncatedText.indexOf(dataKey)) + dataKey + "\"REMOVED BY SERVICEBUILDER\"}";
```